src/player/Player.php

```diff
@@ -515,6 +519,15 @@ public function hasAutoJump() : bool{
 		return $this->autoJump;
 	}
 
+	public function setFlySpeed(float $flySpeed) : void{
```

Comment:
Need probably a value checking, can the flySpeed negative or equal 0 ?

Reply:
You're right. Fly speed can be 0, but not negative